internal/commands/create.go

```diff
@@ -10,11 +10,12 @@ import (
 	"github.com/plexsystems/konstraint/internal/rego"
 
 	"github.com/ghodss/yaml"
-	"github.com/open-policy-agent/frameworks/constraint/pkg/apis/templates/v1beta1"
+	v1 "github.com/open-policy-agent/frameworks/constraint/pkg/apis/templates/v1"
+	v1beta1 "github.com/open-policy-agent/frameworks/constraint/pkg/apis/templates/v1beta1"
```

Comment:
nit: Aliasing the import here doesn't do anything, as the name selected is the same as the original package.

Reply:
removed it for v1beta1 but gofmt defaults to the `v1`.